controllers/topics.controllers.js

```diff
@@ -0,0 +1,19 @@
+const { pullTopics } = require("../models/topics.models");
+
+exports.invalidEndPoint = (req,res,next) => {
```

Comment:
Error handling functions shouldn't be in your controllers folder. They can have their own folder and file to handle errors. Also think about what we want to send back to the user if there is an error - status and a message and how we can test that message is getting posted to our user...